File: 02_ProgrammingFundamentals/_02_DataTypesAndVariables/_01_Lecture/_04_TownInfo.java
package _02_ProgrammingFundamentals._02_DataTypesAndVariables._01_Lecture;

import java.util.Scanner;

public class _04_TownInfo {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input values
        String town = scan.nextLine(), population = scan.nextLine(), area = scan.nextLine();

        // Print result
        System.out.printf("Town %s has population of %s and area %s square km.", town, population, area);
    }
}

File: 24_C#Fundamentals/_02_DataTypesAndVariables/_01_Lecture/_04_TownInfo/_04_TownInfo.cs
using System;

public class _04_TownInfo
{
    public static void Main(string[] args)
    {
        // Read input values
        string town = Console.ReadLine(), population = Console.ReadLine(), area = Console.ReadLine();

        // Print result
        Console.WriteLine($"Town {town} has population of {population} and area {area} square km.");
    }
}
